audio/audio_comp.c

```diff
@@ -51,8 +51,8 @@ struct audio_comp_priv_s
 
   /* This is the contained, low-level audio device array and count. */
 
-  FAR struct audio_lowerhalf_s **lower;
   int count;
+  FAR struct audio_lowerhalf_s **lower;
```

Comment:
could save one pointer space by:
```suggestion
  FAR struct audio_lowerhalf_s *lower[1];
```

Reply:
Done